service/controller/v1/prometheus/prometheus.go

```diff
@@ -106,6 +106,10 @@ var (
 	// PodSDNamespaceLabel is the label applied to the target by Prometheus POD
 	// service discovery that holds the target's Kubernetes namespace.
 	PodSDNamespaceLabel = model.LabelName("__meta_kubernetes_namespace")
+
+	// PodSDGiantswarmServiceTypeLabel is the label applied to the target by Prometheus POD
+	// service discovery that holds the target's Kubernetes pod's label 'giantswarm.io/service_type'.
+	PodSDGiantswarmServiceTypeLabel = model.LabelName("__meta_kubernetes_pod_label_giantswarm_io_service_type")
```

Comment:
on giantswarm managed kiam this label has value `managed` so the customer's pods will have this label  empty, that will be used as a factor to decide if we monitor the pod or not

Reply:
How does this match with the label value?

Reply:
this is just definition of the label prometheus has

the matching happens with this regex https://github.com/giantswarm/prometheus-config-controller/pull/163/files#diff-3b3ca55b1494e07c993f97e0724c5c10R314

which  mtaches againts  kiam pods and empty `__meta_kubernetes_pod_label_giantswarm_io_service_type` label